mixer/pkg/attribute/protoBag.go

```diff
@@ -54,21 +54,23 @@ type ProtoBag struct {
 
 // NewProtoBag creates a new proto-based attribute bag.
 func NewProtoBag(proto *mixerpb.CompressedAttributes, globalDict map[string]int32, globalWordList []string) *ProtoBag {
-	scope.Debugf("Creating bag with attributes: %v", proto)
-
 	// build the message-level dictionary
 	d := make(map[string]int32, len(proto.Words))
 	for i, name := range proto.Words {
 		d[name] = slotToIndex(i)
 	}
 
-	return &ProtoBag{
+	out := &ProtoBag{
 		proto:           proto,
 		globalDict:      globalDict,
 		globalWordList:  globalWordList,
 		messageDict:     d,
 		referencedAttrs: make(map[attributeRef]mixerpb.ReferencedAttributes_Condition, 16),
 	}
+
+	scope.Debugf("Creating bag with attributes: %v", out)
```

Comment:
as soon as we make this change, we'll have some error with the global word list ;).

@geeknoid i think this makes it so we have two scopes for uncompressed attributes (`attributes` and `api`). Is that ok with you?

Reply:
Yeah, I think that's fine.